documentation/functions.md

```diff
@@ -887,6 +891,45 @@ The `NEXT` function returns the value of an expression from a following record i
 - `levels` (optional): optional argument (default `None`) for the same `levels` argument as all other window functions.
 
 
+<!-- TOC --><a name="relsum"></a>
+
+### RELSUM
+
+The `RELSUM` function returns the sum of multiple rows of a singular expression within the same collection, e.g. the global sum across all rows, or the sum of rows per an ancestor of a sub-collection. The arguments:
+
+- `expression`: the singular expression to take the sum of across multiple rows.
+- `levels` (optional): optional argument (default `None`) for the same `levels` argument as all other window functions.
+
+
+<!-- TOC --><a name="relavg"></a>
+
+### RELAVG
+
+The `RELAVG` function returns the average of multiple rows of a singular expression within the same collection, e.g. the global average across all rows, or the average of rows per an ancestor of a sub-collection. The arguments:
+
+- `expression`: the singular expression to take the average of across multiple rows.
+- `levels` (optional): optional argument (default `None`) for the same `levels` argument as all other window functions.
+
+
+<!-- TOC --><a name="relcount"></a>
+
+### RELCOUNT
+
+The `RELCOUNT` function returns the number of non-null records in multiple rows of a singular expression within the same collection, e.g. the count of all non-null rows, or the number of non-null rows per an ancestor of a sub-collection. The arguments:
+
+- `expression`: the singular expression to count the number of non-null entries across multiple rows.
+- `levels` (optional): optional argument (default `None`) for the same `levels` argument as all other window functions.
+
+
+<!-- TOC --><a name="relsize"></a>
+
+### RELSIZE
+
+The `RELSIZE` function returns the number of total records, either globally or the number of sub-collection rows per some ancestor collection. The arguments:
+
+- `levels` (optional): optional argument (default `None`) for the same `levels` argument as all other window functions.
+
```

Comment:
I feel that few examples would be useful since Window functions are advanced stuff.